src/server/routes/requested.js

```diff
@@ -27,25 +28,57 @@ router.get('/', async (req, res) => {
 
 })
 
+const updates = {
+  accept: async function (term) {
+    await db.query(
+      `UPDATE requested SET action = 1 WHERE term = $1`,
+      [term]
+    )
+    
+    return { message: `Accepted term "${term}"` }
+  },
 
-router.post('/:term', async (req, res) => {
+  reject: async function (term) {
+    await db.query(
+      `UPDATE requested SET action = 3 WHERE term = $1`,
+      [term]
+    )
+    
+    return { message: `Rejected term "${term}" :(` }
+  }
+}
+
+async function create (term) {
+  await db.query(
+    `INSERT INTO requested(term, fulfilled, action)
+     SELECT CAST($1 AS VARCHAR),0,0
```

Comment:
I think we would want `SELECT CAST($1 AS VARCHAR),0,1`, to insert with action=1 here